app/helpers/comfy/cms_helper.rb

```diff
@@ -81,5 +81,10 @@ def comfy_paginate(collection)
         paginate collection, :theme => 'comfy'
       end
     end
+
+    def friendly_date(date)
+      date.in_time_zone(Time.zone.name).strftime('%b %e, %l:%M%p')
```

Comment:
Side note, rails has some build in date formatters. Don't know if any of these suit what you're looking for.
http://api.rubyonrails.org/v2.3/classes/ActiveSupport/CoreExtensions/Time/Conversions.html